schema.sql

```diff
@@ -30,4 +30,12 @@ CREATE TABLE species (
   ADD COLUMN species_id BIGINT REFERENCES species (id);
   -- Add column owner_id which is a foreign key referencing the owners table
   ALTER TABLE animals
-  ADD COLUMN owner_id BIGINT REFERENCES owners (id);
\ No newline at end of file
+  ADD COLUMN owner_id BIGINT REFERENCES owners (id);
+
+
+-- Create a table named vets
+CREATE TABLE vets(id SERIAL PRIMARY KEY NOT NULL,name VARCHAR(255), age INT,date_of_graduation date);
+-- Create a table named specialization.
+CREATE TABLE specializations(id SERIAL PRIMARY KEY NOT NULL,vets_name VARCHAR(255), species_name VARCHAR(255));
```

Comment:
Since the `specializations ` table is a `join table` table for `species` and `vets` it has to have the `Primary key` of both tables. So those are the only property this table should have to have.

Consider doing something like this:

```
 species_id int,
 vet_id int
````